src/Common/src/Interop/Unix/System.Native/Interop.MountPoints.FormatInfo.cs

```diff
@@ -143,91 +191,154 @@ private static DriveType GetDriveType(string fileSystemName)
                 case "umview-mod-umfuseiso9660":
                     return DriveType.CDRom;
 
+                case "aafs":
                 case "adfs":
                 case "affs":
+                case "anoninode":
+                case "anon-inode FS":
                 case "apfs":
+                case "autofs":
+                case "balloon-kvm-fs":
+                case "bdevfs":
                 case "befs":
                 case "bfs":
+                case "bpf_fs":
                 case "btrfs":
+                case "btrfs_test":
+                case "cgroup2fs":
+                case "cgroupfs":
+                case "coh":
+                case "cramfs":
+                case "cramfs-wend":
+                case "daxfs":
                 case "drvfs":
                 case "ecryptfs":
+                case "efivarfs":
                 case "efs":
+                case "exofs":
                 case "ext":
                 case "ext2":
                 case "ext2_old":
                 case "ext3":
+                case "ext2/ext3":
                 case "ext4":
                 case "ext4dev":
+                case "f2fs":
                 case "fat":
-                case "fuseblk":
```

Comment:
Why did you move this?  Isn't fuseblk generally how an ntfs file system is reported on Linux?